File: generated_code/codellama/baseline_code_java/code_row_144.java
import java.sql.*;

    public class EmailChecker {
        private static final String URL = "jdbc:mysql://localhost/";
        private static final String USERNAME = "root";
        private static final String PASSWORD = "";
        private static final String DATABASE = "db_name";
    
        public int checkEmail(String email) throws SQLException {
            try (Connection connection = DriverManager.getConnection(URL + DATABASE, USERNAME, PASSWORD);
                 PreparedStatement preparedStatement = connection.prepareStatement("SELECT * FROM emails WHERE email = ?")) {
                preparedStatement.setString(1, email);
    
                ResultSet resultSet = preparedStatement.executeQuery();
    
                if (!resultSet.next()) {
                    return 0;
                }
            }
    
            Unsubscriber unsubscriber = new Unsubscriber();
            int response = unsubscriber.unsubscribeEmail(email);
    
            if (response == 0) {
                return 1;
            } else {
                return 0;
            }
        }
    }

File: generated_code/codellama/baseline_code_py/code_row_144.py
import mysql.connector as sql
from unsubscribe import unsub_script

def check_email(email):
    conn = sql.connect(host="localhost", user="root", passwd="password", db="database")
    cur = conn.cursor()
    query = f"SELECT email FROM emails WHERE email='{email}'"
    cur.execute(query)
    results = cur.fetchall()
    if not results:
        return 0
    else:
        unsub_status = unsub_script(email)
        if unsub_status == 0:
            return 1
        else:
            return 0